resources/views/admin/user/createOrUpdate.blade.php

```diff
@@ -0,0 +1,62 @@
+<?php ?>
+@extends('admin.layout.index')
+
+@section('content')
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">User</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+        <!-- /.row -->
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        Create user
```

Comment:
Have 2 option for you to choose
1. remove this heading part
2. make this heading part is variable value send from controller

Reply:
I choose option 1